### ISSUE TITLE
DSEWorkloads does not work in v1.4.0

### ISSUE THREAD

**Issue by yukim** [datastax/cass-operator#219]
#158 added `dseWorkloads` to CRD, but enabling through this feature does not start DSE properly.

This is because it adds `JVM_EXTRA_OPTS` environmental variables with `-g` and/or `-s` and/or `-k`, and by doing so, JVM crashes when Management API tries to spawn DSE.

DSE Management API swallows stdout when spawning DSE, but with docker you can see it does not start with invalid `JVM_EXTRA_OPTS`:

```
❯ docker run --rm -e DS_LICENSE=accept -e JVM_EXTRA_OPTS="-s -g" datastax/dse-server:6.8.2   
Applying changes to /opt/dse/resources/cassandra/conf/cassandra.yaml ...
done.
Applying changes to /opt/dse/resources/cassandra/conf/cassandra-rackdc.properties ...
done.
Running dse cassandra -f
Unrecognized option: -s
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

I propose to revert the change since DSE Management API seems not support passing args ('-s', '-g', '-k') yet.

**Comment by bradfordcp** [comment]
This feature is a tech-preview. A revert is not in order given it is entirely opt-in, but we should investigate what is causing it to fail start-up.

Running `dse cassandra -f -s` starts just fine within the container. We are validating the behavior of the mgmt api to isolate the problem.

**Comment by jeromatron** [comment]
I've seen others trying to use this who have had similar problems - unable to use the official dse workload mechanism but can still use the workaround of adding jvm args for now.

**Comment by jimdickinson** [comment]
This is fixed in 1.4.1 by #230 and we've got an integration test in place. Please reopen this issue if you can reproduce on a build with that fix.
